layouts/default.vue

```diff
@@ -143,7 +143,7 @@ export default Vue.extend({
           };
         })
     }),
-    items() {
+    items(): Array<Record<string, string>> {
```

Comment:
This should be a custom type

Reply:
@Camc314 Created ``LayoutButton`` type for this.